src/helpers/sort.ts

```diff
@@ -21,6 +21,24 @@ const SORT_TYPES: CellValueType[] = [
   CellValueType.boolean,
 ];
 
+export function cellsSortingCriteria(isAscending: boolean) {
```

Comment:
reading `cellsSortingCriteria(true)` or `cellsSortingCriteria(false)` isn't self-explanatory. Either name the boolean or make it a string param
`cellsSortingCriteria({ ascending: true })` or `cellsSortingCriteria("ascending")`